google/cloud/bigtable/data/_async/client.py

```diff
@@ -236,6 +236,9 @@ def __init__(
                 "is the default."
             )
         self._is_closed = CrossSync.Event()
+        self._disable_background_channel_refresh = bool(
```

Comment:
Add a comment here, explaining that this is a private argument for internal use only